uint16 sonLeft
uint16 sonRight
uint16 sonFront
uint16 sonRear
time timestamp
